concent-builder/configure.yml

```diff
@@ -111,3 +111,10 @@
             state: directory
             owner: "{{ shared_user }}"
             group: "{{ shared_user }}"
+
+        - name: Upload configuration for kubectl
+          copy:
+            src:   "{{ local_secret_dir }}/cloud/config"
```

Comment:
We need some information in the README on how to update this file when it expires or when the IP changes (I see that the IP is hard-coded inside).

Reply:
This file use `gcloudsdk` to update credentials. This is original config, only context names are changes.

Reply:
But you said you changed some stuff in that file. We need information on how to add it to our secrets and how to update it.